README.md

```diff
@@ -305,14 +298,18 @@ Consumer<Throwable> errorHandler = error -> {
     ...
 };
 
-// Create the client using ClientFactory
-ClientFactory clientFactory = new ClientFactory(clientConfig);
-Client client = clientFactory.create(agentCard, consumers, errorHandler);
+// Create the client using the builder
+Client client = Client
+        .from(finalAgentCard)
+        .clientConfig(clientConfig)
+        .addStreamConsumers(consumers)
+        .streamErrorHandler(errorHandler)
+        .build();
 ```
 
 #### Configuring Transport-Specific Settings
 
-Different transport protocols can be configured with specific settings using `ClientTransportConfig` implementations. The A2A Java SDK provides `JSONRPCTransportConfig` for the JSON-RPC transport and `GrpcTransportConfig` for the gRPC transport.
+Different transport protocols can be configured with specific settings using specific `ClientTransportConfig` implementations. The A2A Java SDK provides `JSONRPCTransportConfig` for the JSON-RPC transport and `GrpcTransportConfig` for the gRPC transport.
 
 ##### JSON-RPC Transport Configuration
```

Comment:
The line at the beginning of this section will need to be updated since it currently says:

"For the JSON-RPC transport, if you'd like to use the default JdkA2AHttpClient, no additional configuration is needed..."

But with the current approach, `withJsonRpcTransport` now must be called.

Reply:
Looks like there are still some references to ClientFactory in the README that need to be updated.